src/autotype/mac/AutoTypeMac.cpp

```diff
@@ -488,3 +493,17 @@ void AutoTypeExecutorMac::execKey(AutoTypeKey* action)
     m_platform->sendKey(action->key, false);
     usleep(25 * 1000);
 }
+
+void AutoTypeExecutorMac::execClearField(AutoTypeClearField* action = nullptr)
+{
+    Q_UNUSED(action);
+
+    m_platform->sendKey(Qt::Key_Control, true, true);
+    m_platform->sendKey(Qt::Key_A, true, true);
+    m_platform->sendKey(Qt::Key_Control, false, true);
```

Comment:
>What does it show for the clear field sequence?

![image](https://cloud.githubusercontent.com/assets/107672/24367435/fe19b0da-12f2-11e7-80a2-8b419caa9cd0.png)

The example doesn't seem to follow the same sequence of a real keystroke